exercises/concept/lucians-luscious-lasagna/.meta/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "blurb": "Learn about the basics of Rust by following a lasagna recipe.",
+  "icon": "lasagna",
+  "authors": [
+    "coriolinus",
```

Comment:
@coriolinus I've added you as one of the authors.

Reply:
Thanks!